### PR TITLE
Added list, create and get api docs for ssl certificate.

### PR DESCRIPTION
**Pull request by y0geshdev**
Fixes MC-9443
Code walkthrough : -

Issue
There were no api docs for SSL certificates.

Solution
Added get, list and add ssl certificate api docs.

Test cases
- Manually tested

### CODE REVIEW

File: source/includes/gcp/_ssl_certificates.md [hunk 1]
```diff
@@ -1 +1,159 @@
...
+      "uploadDate": "2019-10-02"
+    },
+    {
+      "certificate": "-----BEGIN CERTIFICATE-----MIIDnDCCAoQCCQDQz2lClG3ipTANBgkqhkiG9w0BAQsFAELMAkGA1UEBhMCIL6BLzjFCyeJ6nMLGwC0v/ptdkT+N4XvfjRjrO1t7B8jOzZy7gqfxElXA5XLpwrWT6AASg9M5IWd87r8Cor/hCuJ25bZbHECAwEAATANBgkqhkiG9w0BAQsFAAOCAQEAFoe0Y8qU0nTBMexn/QBatUVaR88KgmSt8B1vPS0BRyN06jAzzwaJU79yF62qh5mdXXz4dm3mwncrL4v+4HREsmdePwmcMT1f7f7gNRMe2OQfnntIYjkuDvuzxhBgdfqXEr2s
+      -----END CERTIFICATE-----",
+      "creationTimestamp": "2019-10-01T13:33:36.409-07:00",
+      "id": "2973924929495238351",
+      "kind": "compute#sslCertificate",
+      "name": "ssl-root-tcs",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/cmc-gcpenvironment-rjb/global/sslCertificates/ssl-root-tcs",
```
Comment (SeventhOfNine) [inline source/includes/gcp/_ssl_certificates.md]: replace `cmc-gcpenvironment-rjb` with `test-area`

Comment (SeventhOfNine) [reply source/includes/gcp/_ssl_certificates.md]: apply change to all selflinks.

Comment (y0geshdev) [reply source/includes/gcp/_ssl_certificates.md]: Made the above changes.
And also removed garbage certificate and private key body with a message in the bracket indicating what text goes here.

**Review by SeventhOfNine — APPROVED** [review]
Thanks for the changes, looks good to me now.

**Pull request merged**